airbyte-integrations/connectors/source-amazon-seller-partner/acceptance-test-config.yml

```diff
@@ -31,6 +31,27 @@ tests:
 #          "GET_FBA_FULFILLMENT_REMOVAL_SHIPMENT_DETAIL_DATA",
 #          "GET_VENDOR_INVENTORY_HEALTH_AND_PLANNING_REPORT",
 #          "VendorDirectFulfillmentShipping",
+#          "GET_FBA_ESTIMATED_FBA_FEES_TXT_DATA",
```

Comment:
Do not add new streams to `configured_catalog.json`, `configured_catalog_no_empty_streams.json`, and this list. Instead, add one stream per configured catalog file. This way, the test for each such configured catalog runs independently and has a separate timeout, so the tests are more likely to pass. See my comment for file  `configured_catalog_no_empty_streams.json`.

Reply:
done